<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog_report</class>
 <widget class="QDialog" name="Dialog_report">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>710</width>
    <height>644</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>每月結算</string>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>31</x>
     <y>11</y>
     <width>651</width>
     <height>611</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QComboBox" name="comboBox_name"/>
      </item>
      <item>
       <widget class="QComboBox" name="comboBox_date"/>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="label">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>$</string>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QSpinBox" name="spinBox_1">
        <property name="readOnly">
         <bool>true</bool>
        </property>
        <property name="maximum">
         <number>10000</number>
        </property>
       </widget>
      </item>
      <item row="0" column="4">
       <widget class="QSpinBox" name="spinBox_2">
        <property name="readOnly">
         <bool>true</bool>
        </property>
        <property name="maximum">
         <number>10000</number>
        </property>
       </widget>
      </item>
      <item row="0" column="6">
       <widget class="QSpinBox" name="spinBox_3">
        <property name="readOnly">
         <bool>true</bool>
        </property>
        <property name="maximum">
         <number>10000</number>
        </property>
       </widget>
      </item>
      <item row="0" column="8">
       <widget class="QSpinBox" name="spinBox_4">
        <property name="readOnly">
         <bool>true</bool>
        </property>
        <property name="maximum">
         <number>10000</number>
        </property>
       </widget>
      </item>
      <item row="0" column="10">
       <widget class="QSpinBox" name="spinBox_5">
        <property name="readOnly">
         <bool>true</bool>
        </property>
        <property name="maximum">
         <number>10000</number>
        </property>
       </widget>
      </item>
      <item row="0" column="3">
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="0" column="12">
       <widget class="QSpinBox" name="spinBox_6">
        <property name="readOnly">
         <bool>true</bool>
        </property>
        <property name="maximum">
         <number>10000</number>
        </property>
       </widget>
      </item>
      <item row="0" column="11">
       <spacer name="horizontalSpacer_6">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="0" column="9">
       <spacer name="horizontalSpacer_5">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="0" column="5">
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="0" column="1">
       <spacer name="horizontalSpacer_3">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="0" column="7">
       <spacer name="horizontalSpacer_4">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="0" column="13">
       <spacer name="horizontalSpacer_7">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QTableView" name="tableView">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QGridLayout" name="gridLayout_3">
      <item row="0" column="0">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>個別總量</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QSpinBox" name="spinBox_7">
        <property name="readOnly">
         <bool>true</bool>
        </property>
        <property name="maximum">
         <number>10000</number>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QSpinBox" name="spinBox_8">
        <property name="readOnly">
         <bool>true</bool>
        </property>
        <property name="maximum">
         <number>10000</number>
        </property>
       </widget>
      </item>
      <item row="0" column="3">
       <widget class="QSpinBox" name="spinBox_9">
        <property name="readOnly">
         <bool>true</bool>
        </property>
        <property name="maximum">
         <number>10000</number>
        </property>
       </widget>
      </item>
      <item row="0" column="4">
       <widget class="QSpinBox" name="spinBox_10">
        <property name="readOnly">
         <bool>true</bool>
        </property>
        <property name="maximum">
         <number>10000</number>
        </property>
       </widget>
      </item>
      <item row="0" column="5">
       <widget class="QSpinBox" name="spinBox_11">
        <property name="readOnly">
         <bool>true</bool>
        </property>
        <property name="maximum">
         <number>10000</number>
        </property>
       </widget>
      </item>
      <item row="0" column="6">
       <widget class="QSpinBox" name="spinBox_12">
        <property name="readOnly">
         <bool>true</bool>
        </property>
        <property name="maximum">
         <number>10000</number>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="0">
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>個別總收入</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QSpinBox" name="spinBox_13">
        <property name="readOnly">
         <bool>true</bool>
        </property>
        <property name="maximum">
         <number>1000000</number>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QSpinBox" name="spinBox_14">
        <property name="readOnly">
         <bool>true</bool>
        </property>
        <property name="maximum">
         <number>1000000</number>
        </property>
       </widget>
      </item>
      <item row="0" column="3">
       <widget class="QSpinBox" name="spinBox_15">
        <property name="readOnly">
         <bool>true</bool>
        </property>
        <property name="maximum">
         <number>1000000</number>
        </property>
       </widget>
      </item>
      <item row="0" column="4">
       <widget class="QSpinBox" name="spinBox_16">
        <property name="readOnly">
         <bool>true</bool>
        </property>
        <property name="maximum">
         <number>1000000</number>
        </property>
       </widget>
      </item>
      <item row="0" column="5">
       <widget class="QSpinBox" name="spinBox_17">
        <property name="readOnly">
         <bool>true</bool>
        </property>
        <property name="maximum">
         <number>1000000</number>
        </property>
       </widget>
      </item>
      <item row="0" column="6">
       <widget class="QSpinBox" name="spinBox_18">
        <property name="readOnly">
         <bool>true</bool>
        </property>
        <property name="maximum">
         <number>1000000</number>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_5">
      <item>
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>本月收入</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSpinBox" name="spinBox_19">
        <property name="readOnly">
         <bool>true</bool>
        </property>
        <property name="maximum">
         <number>1000000000</number>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
